daemon/bindata.go

```diff
@@ -1,7 +1,9 @@
 // Code generated by go-bindata.
 // sources:
 // ../bpf/COPYING
+// ../bpf/a.sh
 // ../bpf/bpf_lb.c
+// ../bpf/bpf_lxc.S
 // ../bpf/bpf_lxc.c
```

Comment:
For some reason your go-bindata contains `a.sh` and `bpf_lxc.S`. Looks like they are just local files from testing/debugging?